src/content/en/updates/2017/03/custom-scrollbar.md

```diff
@@ -0,0 +1,380 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Custom transforms matrices allow you to build frame-perfect custom scrollbars.
+
+{# wf_updated_on: 2017-03-23 #}
+{# wf_published_on: 2017-03-23 #}
+{# wf_tags: performance #}
+{# wf_featured_image: /web/updates/images/2017/03/custom-scrollbar/poster.jpg #}
+{# wf_featured_snippet: Custom transforms matrices allow you to build frame-perfect custom scrollbars. #}
+
+
+# CSS deep-dive: matrix3d() for a frame-perfect custom scrollbar {: .page-title }
+
+{% include "web/_shared/contributors/surma.html" %}
+
+Scrollbars are one of the remaining bits on the web that are pretty much
+unstylable (I’m looking at you, date picker) and are a thorn in the side of many
+designers. You can use JavaScript to build your own, but that’s expensive, low
+fidelity and can lag behind. In this article we will leverage some
```

Comment:
It’s not janking, as janking means blowing the frame budget and creating overly long frame times. What I mean with “lag behind” is that the position of the scroller’s thumb will be updated with data that is a couple of frames old, making the thumb feel “laggy”.